pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -40,6 +40,8 @@ func (r *ReconcilePerconaServerMongoDB) reconcileCluster(ctx context.Context, cr
 		return api.AppStateReady, nil
 	}
 
+	unsafePsaMode := cr.Spec.UnsafeConf && replset.Arbiter.Enabled && replset.Arbiter.Size == 1 && !replset.NonVoting.Enabled && replset.Size == 2
```

Comment:
@pkeuter please change the variable name in other places too
```suggestion
	// Primary with a Secondary and an Arbiter (PSA)
	unsafePSA := cr.Spec.UnsafeConf && replset.Arbiter.Enabled && replset.Arbiter.Size == 1 && !replset.NonVoting.Enabled && replset.Size == 2
```

Reply:
All done!